assets/source/setup-guide/app/components/SaveSettingsButton.js

```diff
@@ -12,13 +13,33 @@ import {
 	useSettingsDispatch,
 	useCreateNotice,
 } from '../helpers/effects';
+import getContextByPath from '../helpers/get-context-by-path';
 
+/**
+ * Clicking on "… Save changes" button.
+ *
+ * @event wcadmin_pfw_save_changes_button_click
+ */
+
+/**
+ * Save Settings button component
+ *
+ * @fires wcadmin_pfw_save_changes_button_click
+ *
+ * @return {JSX.Element} Rendered element
+ */
 const SaveSettingsButton = () => {
 	const isSaving = useSettingsSelect( 'isSettingsUpdating' );
+	const updatedData = useSettingsSelect( 'getUpdatedData' );
 	const setAppSettings = useSettingsDispatch( true );
 	const createNotice = useCreateNotice();
 
 	const saveSettings = async () => {
+		recordEvent( 'pfw_save_changes_button_click', {
+			updatedData,
+			context: getContextByPath(),
+		} );
+
```

Comment:
❓ I'm thinking whether we need to send this event after successfully saving data to the DB? I know the event name is `pfw_save_changes_button_click` meaning that it's a click event, just wondering which makes more sense from the perspective of people who will grab all the events for analysing.

Reply:
Good question, seems that we are tracking this in [Woo Tracker Snapshot implementation](https://github.com/woocommerce/google-listings-and-ads/blob/develop/src/Tracking/TrackerSnapshot.php): 

Basically we save there the settings the user has in the plugin. Which is a separated issue I'm working on. 

This track is just for track the attempt of saving doesn't matter if it's actually saved or not.

Reply:
Cool, thanks for sharing this:  [Woo Tracker Snapshot implementation](https://github.com/woocommerce/google-listings-and-ads/blob/develop/src/Tracking/TrackerSnapshot.php). Agree with this track is just for clicking and good to know we will be having the track of saving soon. 👍